src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40 * days;
+  const shortPeriod = 3;
+  const longPeriod = 7;
+  const discountSmall = 20;
+  const discountBig = 50;
+
+  if (days < shortPeriod) {
+    return rentCost;
+  }
+
+  if (days >= shortPeriod && days < longPeriod) {
```

Comment:
This condition is too complex, you can simplify it 👇
```suggestion
  if (days < longPeriod) {
```